src/airport.js

```diff
@@ -25,9 +29,11 @@ class Airport {
   };
 
   isPlaneInAirport = plane => {
-    if (plane instanceof Plane) !this.airportPlanes.includes(`test1`);
-
-    if (plane instanceof Plane && !this.takeOffPlane());
+    if ((plane instanceof Plane) && !this.airportPlanes.includes(plane)) {
+      return true
+    } else {
+      return false;
+    }
   };
 }
 
```

Comment:
I feel like my airport class is quite messy and perhaps there are too many methods and if statements.  I'm not happy with the isPlaneInAirport method and know there must be a simpler and cleaner way to code this without returning true or false. Perhaps sticking to array methods and removing the ifs? I think its something I need to look into.